File: src/main/java/info/dddeurope/lab/app/domain/UserAggregate/Age.java
package info.dddeurope.lab.app.domain.UserAggregate;

import lombok.AllArgsConstructor;
import lombok.Data;

@Data
@AllArgsConstructor
public class Age {
    int age;
    
    boolean isUnder(Age age) {
        return this.getAge() < age.getAge();
    }
}

File: DotNet/App/Domain/UserAggregate/Age.cs
namespace App.Domain.UserAggregate
{
    public class Age
    {
        public int AgeInYears { get; }

        public Age(int ageInYears)
        {
            AgeInYears = ageInYears;
        }

        public bool IsUnder(Age age)
        {
            return this.AgeInYears < age.AgeInYears;
        }
    }
}